src/pfs/pfs.proto

```diff
@@ -284,6 +294,31 @@ message DeleteBranchRequest {
   bool force = 2;
 }
 
+message CreateProjectRequest {
+  Project project = 1;
+  string description = 2;
+  bool update = 3;
+}
+
+message CreateProjectResponse {}
+
+message InspectProjectRequest {
+  Project project = 1;
+}
+
+message InspectProjectResponse {
```

Comment:
This seems like it should be consistent with the other inspect rpcs which just return the Pachyderm object's info.